working-groups/performance-testing/README.md

```diff
@@ -0,0 +1,92 @@
+# HPSF Benchmarking Working Group
+
+Benchmarking WG aims to bring together projects and organizations 
+to create an automated way to 
+continuously test the performance of evolving HPC software stacks, 
+ensuring that the components work and perform well.
+Functionality testing and performance regression testing are both in scope.
+Parts of the software stack we would like to test:
+* Compilers
+* Runtimes and programming models
+* MPI and other communication libraries
+* Math libraries
+* HPC applications
+  
+
+## Leads
+
+* Olga Pearce ([@pearce8](https://github.com/pearce8)])
+* ???
+
+
+## Meetings
+
+* The working group meets every **TBD** weeks. [meeting notes](https://docs.google.com/document/d/13edPtYo5hPpbFg7PO71v8RMGJByJRb1fUY1JPuwshkA/edit?usp=sharing)
+* Zoom coordinates are **TBD**.
+* To get involved, join the [Slack channel](https://hpsf.slack.com/archives/C08NKKL97NZ)
+
+
+## Motivation for launch
+
+* There is not a group that is consolidating and maintaining open source benchmarks
+* Each datacenter is for themselves
+* Testing/benchmarking of the HPC software stack is intermittent at best
+* We want to figure out, as a working group, how we can pull together resources to support and maintain continuous testing
+
+
+## Initial objectives
+
+* First goal: Run [insert benchmark here] on [insert system here]
+* Identify open source benchmarks of general interest
+* Work towards running the benchmarks in a sustainable way on a few systems
+
+
+## Anticipated activities
+
+* Community contributed catalogued library of HPC tests and benchmarks
+  * Single component tests
+  * Performance benchmarks
+  * Proxy applications
+  * Large applications
+
+* Software stack representation: sharing methodologies, creating best practices
+  * Introspection
+  * Reproducibility
+  * Changing one component version at a time to isolate failures
```

Comment:
Defining good measurement/instrumentation granularity in a benchmark to make it easy to observe what part of a library/application might have improved/regressed. Nothing more frustrating than having a single number represent thousands of lines of code.